deposit/bytecode-verification/.build/.kevm.rev

```diff
@@ -1 +0,0 @@
-652576cb753656da51a25cea754d38c3d59470cd
```

Comment:
update? https://github.com/runtimeverification/verified-smart-contracts/blob/master/deposit/bytecode-verification/Makefile#L1

Reply:
BTW, please do not remove this local .kevm.rev file until the deposit verification is completed.  We are still frequently updating the kevm version.

Reply:
Fixed all. Please review.

Reply:
For Haskell support I guess I'll have to implement --concrete-rules in front-end.

Reply:
Rolled back updates deposit/kevm.rev - some failures.